roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/data/RoqFrontMatterDataProcessor.java

```diff
@@ -45,6 +48,12 @@ void prepareData(HttpBuildTimeConfig httpConfig,
             final JsonObject data = mergeParents(item, byKey);
             final String link = Link.pageLink(config.rootPath(), data.getString(LINK_KEY, DEFAULT_PAGE_LINK_TEMPLATE),
                     new Link.PageLinkData(item.info().baseFileName(), item.info().date(), item.collection(), data));
+
+            if (!item.aliases().isEmpty()) {
+                aliases.produce(new RoqFrontMatterAliasesBuildItem(
+                        item.aliases(), link));
```

Comment:
Let's iterate on the aliases and create one item per alias and use the Link.pageLink with the rootpath (same a above) and with the same PageLinkData. This will allow templating of the aliases (and add the Roq rootPath).